docs/source/datastore.rst

```diff
@@ -14,12 +14,12 @@ are resolved on rule evaluation.
 Storing and Retrieving Key Value Pairs from CLI
 -----------------------------------------------
 
-Create new key value pairs.
+Set a value of a key value pair.
 
 ::
 
-    st2 key create os_keystone_endpoint http://localhost:5000/v2.0
```

Comment:
This leads to inconsistency with rest of our CLI. Have you tried playing with the CLI to create rules, actions etc and also with keys in roughly the same session. If a user needs to  learn new syntax to handle key management it could be annoying.